extensions/amp-jwplayer/0.1/amp-jwplayer.js

```diff
@@ -371,6 +407,41 @@ class AmpJWPlayer extends AMP.BaseElement {
     return placeholder;
   }
 
+  /**
+   * @private
+   */
+  onSetup_() {
+    const {element} = this;
+    const configAttributes = getDataParamsFromAttributes(
+      element,
+      null,
+      /^config(.+)/
+    );
+    const configJSON = element.getAttribute('data-config-json');
+    const config = tryParseJson(configJSON) || {};
+
+    Object.keys(configAttributes).forEach((attr) => {
```

Comment:
@alanorozco Thoughts? This appears to be a filter + copy.

Reply:
It looks like `data-config-json` and `data-config-json-*` are equivalent with the latter overriding specific properties in the former. It's the same case with `data-ad-cust-params` and `adMacro*`

I would suggest that we stick to a single API, either a single attribute with all properties, or single attributes setting every property. But not both.

Reply:
There are actually only three options for the `data-config-*` attribute, `data-config-skin`, `data-config-plugin`, and `data-config-json`, so `data-config-json-*` attributes are not expected. They follow that `data-config-*` naming convention because they are player config settings.

`getDataParamsFromAttributes` was used to grab those first two together, but they can instead be explicitly grabbed like this:
```
const configSkin = element.getAttribute('data-config-skin');
const configPlugin = element.getAttribute('data-config-plugin');
```
The `data-ad-*` attributes are similar in that they are player ad settings. So `data-ad-cust-params` is standalone, but multiple adMacros can be set with `data-ad-macro-*`.

All these attributes are set on the config object to be sent together to the player, but that object can be renamed or a separate one can be used to make it more clear.
Open to any thoughts and ideas.